BDArmory/Distribution/GameData/BDArmory/Parts/ArmorPlate/TripanelArmor.cfg

```diff
@@ -63,9 +52,9 @@ tags = armor Armo Ship Afv panel
 	{
 	  	name = BDAdjustableArmor
 		ArmorTransformName = ArmorTransform
-		Node1Name = N1
-		Node2Name = N2	
-		isTriangularPanel = True
+		isTriangularPanel = true
+		TriangleType = Right
```

Comment:
Can we also get an isosceles triangle to avoid having to use pairs of right triangles?

Reply:
Yes, isosceles would be doable.

Reply:
Actually, you can do this already with the new equilateral tri armor panel. Did you mean a Scalene Triangle? (which is also doable)

Reply:
No, isosceles was what I meant. Does this mean that the equilateral triangle armor panel is actually isosceles then?
I.e., you can adjust the base and height independently (for equilateral, you've only got a single length that can be adjusted)?

Reply:
yes. length/width are independently scalable